wvd-sh/arm-template-customization/script.ps1

```diff
@@ -0,0 +1,15 @@
+﻿##############################################################
+#  Run the Virtual Desktop Optimization Tool (VDOT)
+##############################################################
+# https://github.com/The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool
+
+# Download VDOT
+$URL = 'https://github.com/The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool/archive/refs/heads/main.zip'
```

Comment:
This script is not valid since paths are not working. We should add here more workable script

Reply:
@dhrump the feature in the portal is called "Post Update Custom Configuration".  I believe it makes sense to at least keep that text so customers can easily identify the solution.  The VDOT path works for everyone else I've shared my code with, internally and externally.  Not sure why you're having a problem.  VDOT is a common post configuration with AVD and I would hate to see the code updated with a solution that isn't helpful.

Reply:
interesting. i was not able to access the path